README.md

```diff
@@ -109,7 +114,7 @@ After rebooting Jetson, the D457 driver should work.
 
 **NOTE**
 
-- Each JetPack version's kernel may be different, the user needs to change the kernel version in file names and paths accordingly, for example for JetPack 4.6.1 the version is `4.9.253-d457`.
+- Each JetPack version's kernel may be different, the user needs to change the kernel version in file names and paths accordingly, for example for JetPack 4.6.1 the version is `4.9.253-d457` or `4.9.253-tegra`, depands on the build method applied.
```

Comment:
typo
"......dep~~a~~**e**nd~~s~~**ing** on the build method applied"

Reply:
Fixed